dist-c/microvium.h

```diff
@@ -189,21 +195,21 @@ void mvm_free(mvm_VM* vm);
  * MVM_E_UNCAUGHT_EXCEPTION and the exception value will be put into
  * `out_result`
  */
-mvm_TeError mvm_call(mvm_VM* vm, mvm_Value func, mvm_Value* out_result, mvm_Value* args, uint8_t argCount);
+MVM_EXPORT mvm_TeError mvm_call(mvm_VM* vm, mvm_Value func, mvm_Value* out_result, mvm_Value* args, uint8_t argCount);
 
-void* mvm_getContext(mvm_VM* vm);
+MVM_EXPORT void* mvm_getContext(mvm_VM* vm);
 
-void mvm_initializeHandle(mvm_VM* vm, mvm_Handle* handle); // Handle must be released by mvm_releaseHandle
-void mvm_cloneHandle(mvm_VM* vm, mvm_Handle* target, const mvm_Handle* source); // Target must be released by mvm_releaseHandle
-mvm_TeError mvm_releaseHandle(mvm_VM* vm, mvm_Handle* handle);
-static inline mvm_Value mvm_handleGet(const mvm_Handle* handle) { return handle->_value; }
-static inline void mvm_handleSet(mvm_Handle* handle, mvm_Value value) { handle->_value = value; }
+MVM_EXPORT void mvm_initializeHandle(mvm_VM* vm, mvm_Handle* handle); // Handle must be released by mvm_releaseHandle
+MVM_EXPORT void mvm_cloneHandle(mvm_VM* vm, mvm_Handle* target, const mvm_Handle* source); // Target must be released by mvm_releaseHandle
+MVM_EXPORT mvm_TeError mvm_releaseHandle(mvm_VM* vm, mvm_Handle* handle);
+MVM_EXPORT static inline mvm_Value mvm_handleGet(const mvm_Handle* handle) { return handle->_value; }
```

Comment:
I spoke too soon.  I didn't notice that you'd added the macro on these as well.  We don't want it on static inline functions because they exist in the caller's compilation unit, not in the library.

Reply:
My mistake. Let me know if https://github.com/coder-mike/microvium/pull/55 doesn't fix this.